src/internal/m365/collection/exchange/contacts_container_cache.go

```diff
@@ -93,8 +94,10 @@ func (cfc *contactContainerCache) Populate(
 	baseID string,
 	baseContainerPath ...string,
 ) error {
+	logger.Ctx(ctx).Info("populating container cache")
+
 	if err := cfc.init(ctx, baseID, baseContainerPath); err != nil {
-		return clues.Wrap(err, "initializing")
+		return clues.WrapWC(ctx, err, "initializing")
```

Comment:
`init` (and, below, `enumerateContainers` and `populatePaths`) all take in Ctx.  Do they not attach the context to the error already?

Reply:
hmmm seems like they do